### ISSUE TITLE
Run tests in random order

### ISSUE THREAD

**Issue by gforcada** [plone/jenkins.plone.org#247]
I created this job: https://jenkins.plone.org/view/Experimental/job/Shuffle%20tests (it boils down to `bin/test --shuffle`).

On the first run we manage to create a stackoverflow! https://jenkins.plone.org/view/Experimental/job/Shuffle%20tests/1/console

On the second run only 4 test failures: https://jenkins.plone.org/view/Experimental/job/Shuffle%20tests/2/console

I set it to run once a week.

Is there anything else that we should do here? Send email to someone? :thinking: 

/cc @Rotonen @tisto @plone/testing-team

**Comment by Rotonen** [comment]
Yes, someone should attend to everything it finds. So we need to find someone interested in the quest and to have that person notified.

As for the run itself, it is doing exactly what that one should do.
